### ISSUE TITLE
Cairo and R can't be build

### ISSUE THREAD

**Issue by luispuerto** [sethrfore/homebrew-r-srf#38]
Cairo returns

```shell
Last 15 lines from /Users/lpuerto/Library/Logs/Homebrew/cairo/01.configure:
checking for sys/shm.h... yes
checking whether shmctl IPC_RMID allowes subsequent attaches... no
checking for X11/extensions/XShm.h... no
checking for X11/extensions/shmproto.h... yes
checking for X11/extensions/shmstr.h... yes
checking whether cairo's Xlib surface backend feature could be enabled... yes
checking for cairo's Xlib Xrender surface backend feature...
checking for xlib_xrender... no
checking for X11/extensions/Xrender.h... no
checking for XRenderCreateSolidFill... no
checking for XRenderCreateLinearGradient... no
checking for XRenderCreateRadialGradient... no
checking for XRenderCreateConicalGradient... no
checking whether cairo's Xlib Xrender surface backend feature could be enabled... no (requires  https://freedesktop.org/Software/xlibs)
configure: error:  Xlib Xrender surface backend feature could not be enabled
```

Y R returns 

```shell 
Last 15 lines from /Users/lpuerto/Library/Logs/Homebrew/r/01.configure:
checking whether iswctype exists and is declared... yes
checking for wctrans_t... yes
checking for mbstate_t... yes
checking for ICU... no
checking for ucol_open in -licucore... yes
checking for X... libraries , headers
checking for gethostbyname... yes
checking for connect... yes
checking for remove... yes
checking for shmat... yes
checking for IceConnectionNumber in -lICE... no
checking X11/Intrinsic.h usability... no
checking X11/Intrinsic.h presence... no
checking for X11/Intrinsic.h... no
configure: error: --with-x=yes (default) and X11 headers/libs are not available
```

Perhaps it's related to #37. 

Note that I'm on Big Sur 11.1, on the master formula and `libx11` and `XQuartz` are installed.

**Comment by luispuerto** [comment]
Seems that some of the X headers can't be found.

**Comment by sethrfore** [comment]
Is this based on the most recent update to the master branch?

**Comment by sethrfore** [comment]
Both r and cairo build fine (mostly, TclTk is broken again) on my system. Here is my brew config output

HOMEBREW_VERSION: 2.6.2-207-gf47c1ea
ORIGIN: https://github.com/Homebrew/brew
HEAD: f47c1eae3256fe263b3adf60e8e437b97bb5eb7d
Last commit: 6 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 386111a04c8f09f72c60548e519a3e27fade9c34
Core tap last commit: 43 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.OGP4zjC5gR/org.macosforge.xquartz:0
HOMEBREW_EDITOR: /usr/local/bin/subl
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_NO_ANALYTICS: set
HOMEBREW_NO_EMOJI: set
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit kabylake
Clang: 12.0 build 1200
Git: 2.29.2 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
Java: 1.8.0_275
macOS: 11.1-x86_64
CLT: 12.3.0.0.1.1607026830
Xcode: 12.3
XQuartz: 2.7.11 => /opt/X11

**Comment by luispuerto** [comment]
I can recheck everything and return here, but I think it's based on the last update.

**Comment by luispuerto** [comment]
@sethrfore no, it wasn't based on the most recent update of master. I think you updated at the same time I had the problem. Now it's building perfectly. 

fixed with 591c6ec78680ea5cf4585265467510ea1f746dd3
